### DISCUSSION TITLE
Stale dates on your commits page

### DISCUSSION THREAD

**Discussion by g-busch** [tannerhelland/PhotoDemon#429] [Q&A]
Why do I often get a very old result on your main page here, sometimes as old as 2 or 3 weeks, when it's actually been updated recently?

For example, today, when looking for your latest commits I found myself back on July 15, then had to reload the page half a dozen times to arrive at the latest, which was September 6, yesterday's date. Could it be a GitHub server issue? I always clear my cache and cookies when I start a new browser session.

**Comment by tannerhelland** [comment]
Hello @g-busch .  I'm afraid I don't have any useful feedback here!

I know GitHub itself is always up-to-date because that's how I sync my work across multiple PCs.   If I update code from a laptop, for example, I can immediately sync those changes through GitHub to a desktop PC.  This works 100% reliably and I use it daily (as do many other developers, I imagine).

The dedicated commit page for any project - available by clicking the commit count on a project's main page - will always be up-to-date, assuming there is no local caching issue.  For example, PhotoDemon's full commit list is always accessible via this direct link:

https://github.com/tannerhelland/PhotoDemon/commits/main

Please let me know if the problem persists and I can try to dig deeper.  I exclusively use Firefox and Safari, but if you're using another browser perhaps there is a setting we can toggle to improve the page update behavior for you.
